common/decisions/00_major_decisions_british_india.txt

```diff
@@ -30,6 +30,81 @@ form_british_raj_decision = {
 		}
 	}
 
+	is_valid = {
+		AND = {
+			completely_controls_region = custom_british_raj
+		}
+	}
+
+	is_valid_showing_failures_only = {
+		is_landed = yes
+		is_capable_adult = yes
+		is_imprisoned = no
+		is_at_war = no
+	}
+
+	effect = {
+		gain_heroic_legend_seed_tooltip_effect = yes
+		custom_tooltip = form_british_raj_decision_effect_tooltip
+		
+		hidden_effect = {
+			create_title_and_vassal_change = {
+				type = created
+				save_scope_as = change
+				add_claim_on_loss = no
+			}
+			
+			title:e_british_raj = {
+				change_title_holder = {
+					holder = root
+					change = scope:change
+				}
+			}
+			resolve_title_and_vassal_change = scope:change
+		}
+	}
+
+	ai_potential = {
+		always = yes
+	}
+
+	ai_will_do = {
+		base = 100
+	}
+}
+
+form_greater_british_raj_decision = {
+	picture = {
+		reference = "gfx/interface/illustrations/decisions/decision_dynasty_house.dds"
+	}
+	desc = form_greater_british_raj_decision_desc
+	selection_tooltip = form_greater_british_raj_decision_tooltip
+	decision_group_type = major
+
+	ai_check_interval = 120
+
+	is_shown = {
+		is_ruler = yes
+		is_playable_character = yes
+		highest_held_title_tier > 3
+
+		root = {
+			OR = {
+				culture = culture:irish
+				culture = culture:scottish
+				culture = culture:gaelic
+				culture = culture:english
+				culture = culture:anglo_saxon
+				culture = culture:welsh
+				culture = culture:cornish
+			}
+		}
+
+		NOT = {
+			has_title = title:e_british_raj
+		}
+	}
```

Comment:
The is_shown for the greater raj decision should only be shown if the character has the British Raj title (you'll never wanna see both decisions at the same time)

Aka remove the NOT wrapper

Reply:
You appear to have removed the entire NOT codeblock...
The NOT wrapper is not everything, it's the "wrapper", aka:
```
NOT = { # <-- This is the wrapper
   ...anything # <-- This is the content
}
```

You need the title content since you're basically saying, the character need to have the British Raj title for this decision to appear.

Your update has made it so any Britannic culture character will see this decision.